agents-api/tests/test_task_routes.py

```diff
@@ -120,11 +122,12 @@ def _(make_request=make_request, task=test_task):
     assert response.status_code == 200
 
 
+# FIXME: This test is failing
```

Comment:
That might happen because the transition created in `test_transition` does not belong to the execution created in `test_execution`

Reply:
No this is happening because of an invalid transition. It seems that at the time of running this fixture, there is already a transition with type "cancelled". And this fixture transition is of type "step".

Since "cancelled"->"step" is an invalid transition, this query fails